server/middlewares/CommentValidation.js

```diff
@@ -0,0 +1,83 @@
+// Model modules
+import models from '../db/models';
+
+const { Comment, User } = models;
+
+/**
+ * @class CommentValidation
+ */
+class CommentValidation {
+  /**
+   * @static
+   * @param {object} request
+   * @param {object} response
+   * @param {function} next
+   * @returns {object} object
+   * @memberof CommentValidation
+   */
+  static validateComment(request, response, next) {
+    const { commentBody } = request.body;
+    console.log('============>>>>', commentBody);
```

Comment:
All console.logs should be removed before pushing to the repo such as line 20 which reads: 
`console.log('============>>>>', commentBody);`
A great way to do this is to use `command + f` buttons to search for and remove all occurrences of it.